src/services/coronastatusService.js

```diff
@@ -3,7 +3,8 @@ import { EventTargetShim } from '../utils/EventTargetShim.js';
 class CoronaStatusService extends EventTargetShim {
   async getCoronaStatus(forceRefresh) {
     if (forceRefresh || !this.__coronastatus) {
-      this.__coronastatus = await (await fetch(new URL('../../data/coronastatus.json', import.meta.url))).json();
+      this.__coronastatus = fetch(new URL('../../data/coronastatus.json', import.meta.url));
+      this.__coronastatus = await (await this.__coronastatus).json();
       this.dispatchEvent(new Event('change'));
     }
     return this.__coronastatus;
```

Comment:
This function now returns either the raw fetch request or the parsed JSON. The first case is incorrect.

What about:

```js
if (forceRefresh || !this.__coronastatus) {
  this.__coronastatus = fetch(new URL('../../data/coronastatus.json', import.meta.url)).then(r => r.json()) 
  await this.__coronastatus;   
  this.dispatchEvent(new Event('change'));
}
return this.__coronastatus;
```

Reply:
ah yes, you're right, I fixed it